crates/cxx-qt-gen/test_outputs/passthrough_and_naming.rs

```diff
@@ -780,6 +780,9 @@ cxx_qt::static_assertions::assert_eq_size!(
     cxx_qt::signalhandler::CxxQtSignalHandler<SecondObjectCxxQtSignalClosurepropertyNameChanged>,
     [usize; 2]
 );
+fn my_function(self: &SecondObject, param: i32) {
+    self.rust().my_function(param)
+}
```

Comment:
this generated code still needs to end up being wrapped in a impl.

So it should be

```rust
impl ffi::SecondObject {
    fn my_function(self: &SecondObject, param: i32) {
        self.rust().my_function(param)
    }
}
```

As this is effectively a shorthand so the developer doesn't need to write that, as it's just a pass through